daprdocs/content/en/operations/monitoring/tracing/datadog.md

```diff
@@ -0,0 +1,36 @@
+---
+type: docs
+title: "How-To: Set up Datadog for distributed tracing"
+linkTitle: "Datadog"
+weight: 1000
+description: "Set up Datadog for distributed tracing"
+---
+
+Dapr captures metrics and traces that can be sent directly to Datadog through the OpenTelemetry Collector Datadog exporter.
+
+## Configure Tracing in Dapr with the OpenTelemetry Collector and Datadog
+
+You can configure Dapr to create traces for each application in your Kubernetes cluster and collect those traces in Datadog, using the OpenTelemetry Collector Datadog exporter.
+
+Follow [the Dapr docs for setting up the OpenTelemetry Collector](https://docs.dapr.io/operations/monitoring/tracing/otel-collector/open-telemetry-collector/#setting-opentelemetry-collector) and:
+
+  1. Add your Datadog API key to the `./deploy/opentelemetry-collector-generic-datadog.yaml` file in the `datadog` exporter configuration section:
```

Comment:
```suggestion
1. Add your Datadog API key to the `./deploy/opentelemetry-collector-generic-datadog.yaml` file in the `datadog` exporter configuration section:
```

Also, need to be more clear on where in the yaml to put this - maybe include more of the yaml file in the snippet below for context?

Reply:
Will find this out and add it 👍